TestBots/packages.config

```diff
@@ -3,5 +3,5 @@
   <package id="MSTest.TestAdapter" version="2.2.7" targetFramework="net48" />
   <package id="MSTest.TestFramework" version="2.2.7" targetFramework="net48" />
   <package id="Newtonsoft.Json" version="13.0.1" targetFramework="net48" />
-  <package id="TricksterBotClasses" version="1.0.1" targetFramework="net48" />
+  <package id="TricksterBotClasses" version="1.0.2" targetFramework="net48" />
```

Comment:
@johnson520 looks like this is failing CI because 1.0.2 hasn't been published to NuGet yet. We'll need to get that live before this can merge.